packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+
+###### Request parameters
+
+
+| Parameter       | Description                                | Param Type   | DataType   | Required |
+| :-------------- | :----------------------------------------- | :----------- | :--------- | :------- |
+| groupId         | `id` of an admin Group                     | URL          | String     | TRUE     |
+| roleId          | `id` of a Role                             | URL          | String     | TRUE     |
+| targetGroupId   | `id` of Group target for role assignment   | URL          | String     | TRUE     |
+
+###### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/JBCUYUC7IRCVGS27IFCE2SKO/targets/groups/00gsrhsUaRoUib0XQ0g3"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### App administrator role App targets
+
+Assign an admin role to a subset of Apps to grant the admin permission to manage all instances of those Apps. Targeted Apps are Okta catalog Apps, and you can assign App targets for these Apps to an admin role regardless of whether a specific instance of the App has been created. For example, there can be multiple configurations of one Okta catalog App, such as Salesforce or Facebook. When you add a Salesforce or Facebook App as a target, that grants the admin permission to manage all instances of Salesforce or Facebook Apps and create new instances of them.
+
+#### List App targets for an App administrator role
+
+##### List App targets for an App administrator role given to a User
+
+<ApiOperation method="get" url="/api/v1/users/${userId}/roles/${roleId}/targets/catalog/apps" />
+
+Lists all App targets for an `APP_ADMIN` Role assigned to a User
+
+###### Request parameters
+
+
+| Parameter   | Description                                                    | Param Type   | DataType   | Required |
+| :---------- | :------------------------------------------------------------- | :----------- | :--------- | :------- |
+| after       | Specifies the pagination cursor for the next page of targets   | Query        | String     | FALSE    |
+| limit       | Specifies the number of results for a page (default is 20)     | Query        | Number     | FALSE    |
+| roleId      | `id` of a Role                                                 | URL          | String     | TRUE     |
+| userId      | `id` of a User                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
```

Comment:
Treat the page cursor as an opaque value: obtain it through the next link relation. > Treat the page cursor as an opaque value. You can obtain it through the next link relation.